### PR TITLE
Fix error on empty items array

### PR DESCRIPTION
**Pull request by cvarley100**
If you pass an empty array to the `items` prop, the component will throw an error. This is due to `getSelectedItem` method returning undefined for `selectedItem`.

### CODE REVIEW

**Comment by adamfellon** [comment]
I get this `TypeError: requested keys of a value that is not an object` on android specifically, do you know if this addresses that issue as well?

**Comment by lfkwtz** [comment]
Closes #161

File: src/index.js [hunk 1]
```diff
@@ -106,7 +106,7 @@ export default class RNPickerSelect extends PureComponent {
             idx = 0;
         }
         return {
-            selectedItem: items[idx],
+            selectedItem: items[idx] || '',
```
Comment (lfkwtz) [inline src/index.js]: make this `|| {}` to more closely match the signature of an individual item

**Comment by lfkwtz** [comment]
@cvarley100 want to make this change?

**Pull request closed without merging**